### PR TITLE
Meta: Support a different build configuration for WPT.sh

### PR DESCRIPTION
**Pull request by shannonbooth**
I generally use the Debug build configuration instead of the default one. While I don't use `ladybird.sh`, I do use `WPT.sh` - so change the script to support a different build folder in line with `ladybird.sh`'s implementation.

### CODE REVIEW

File: Meta/shell_include.sh [hunk 1]
```diff
@@ -60,3 +60,26 @@ ensure_ladybird_source_dir() {
...
+        "Sanitizer")
+            BUILD_DIR="${LADYBIRD_SOURCE_DIR}/Build/ladybird-sanitizers"
+            ;;
+        *)
+            echo "Unknown BUILD_PRESET: '$1'" >&2
+            exit 1
+            ;;
+    esac
+
+    echo "${BUILD_DIR}"
```
Comment (AtkinsSJ) [inline Meta/shell_include.sh:84]: Is this echo left over from testing the script?

Comment (shannonbooth) [reply Meta/shell_include.sh:84]: It's for the caller to do the `BUILD_DIR=` part: `BUILD_DIR=$(get_build_dir "$BUILD_PRESET")`

Comment (AtkinsSJ) [reply Meta/shell_include.sh:84]: Ah ok, my shell-noobishness is showing. :sweat_smile:

Comment (shannonbooth) [reply Meta/shell_include.sh:84]: I am no expert myself honestly!

**Review by tcl3 — APPROVED** [review]

**Pull request merged**